libimage/runtime.go

```diff
@@ -515,20 +516,29 @@ func (r *Runtime) imageReferenceMatchesContext(ref types.ImageReference, options
 		return false, err
 	}
 
+	writeMessage := func(msg string) {
+		if writer == nil {
+			logrus.Debugf("WARNING: %s", msg)
+		} else {
+			fmt.Fprintf(writer, "WARNING: %s\n", msg)
```

Comment:
Should this be also `logrus.Warnf`

Reply:
No, I want this on the `writer`. This way, it'll be displayed on the remote clients as well.